packages/validate/src/ValidateMixin.js

```diff
@@ -479,15 +480,19 @@ export const ValidateMixin = dedupeMixin(
         return JSON.stringify(result) !== JSON.stringify(prevResult);
       }
 
-      __isEmpty(v) {
+      __isEmpty() {
         if (typeof this._isEmpty === 'function') {
-          return this._isEmpty(v);
+          return this._isEmpty();
```

Comment:
Maybe we should pass the modelValue for backw compatibility?

Reply:
Done 👍